src/util.py

```diff
@@ -260,12 +265,160 @@ def xr_add_cyclic_longitudes(da, coord):
 
     return new_da
 
+
+def plotCLabels(da, contours, transform, ax, proj, clabel_locations=[], fontsize=12, whitebbox=False, horizontal=False):
+
+    """
+    Utility function to plot contour labels by passing in a coordinate to the clabel function.
+    This allows the user to specify the exact locations of the labels, rather than having matplotlib
+    plot them automatically.
+
+    Args:
+        da: (:class:`xarray.DataArray`):
+            Xarray data array containing the lat, lon, and field variable data values.
+        contours (:class:`cartopy.mpl.contour.GeoContourSet`):
+            Contour set that is being labeled.
+        transform (:class:`cartopy._crs`):
+            Instance of CRS that represents the source coordinate system of coordinates.
+            (ex. ccrs.Geodetic()).
+        ax (:class:`matplotlib.pyplot.axis`):
+            Axis containing the contour set.
+        proj (:class:`cartopy.crs`):
+            Projection 'ax' is defined by.
+            This is the instaance of CRS that the coordinates will be transformed to.
+        clabel_locations (:class:`list`):
+            List of coordinate tuples in GPS form (lon in degrees, lat in degrees)
+            that specify where the contours with regular field variable values should be plotted.
+        fontsize (:class:`int`):
+            Font size of contour labels.
+        whitebbox (:class:`bool`):
+            Setting this to "True" will cause all labels to be plotted with white backgrounds
+        horizontal (:class:`bool`):
+            Setting this to "True" will cause the contour labels to be horizontal.
+    Returns:
+        cLabels (:class:`list`):
+            List of text instances of all contour labels
+    """
```

Comment:
Can we add a link to one of the example plots that makes use of this convenience function?